packages/@okta/i18n/src/properties/login.properties

```diff
@@ -280,6 +280,17 @@ recovery.sms = SMS
 recovery.call = Voice Call
 recovery.smsOrCall = SMS or Voice Call
 
+# Email enrollment and verification
+send.email.code.save = Send me the code
```

Comment:
nit: it is easier to identify strings if they have some common prefix, maybe something as simple as `email.*` or `email.factor.*`